src/component/feature/FeatureToggleList/FeatureToggleListActions/FeatureToggleListActions.tsx

```diff
@@ -73,7 +79,7 @@ export const FeatureToggleListActions: VFC<IFeatureToggleListActionsProps> = ({
                 data-loading
             />
             <ConditionallyRender
-                condition={uiConfig.flags.P}
+                condition={uiConfig.flags.P && !inProject}
```

Comment:
I don't want to spend that much time on the actual FeatureToggleList component since we are going to make some changes here soon, but the more we can avoid having a component for two different use cases, the easier it will be to work with. We should strive to lift these things up into the parent components where possible and use composition to change elements that are more deeply nested.